ci/edmtool.py

```diff
@@ -305,13 +307,23 @@ def docs(runtime, toolkit, environment):
     execute(commands, parameters)
     click.echo("Done installing documentation tools")
 
+    api_path = os.path.join("docs", "source", "api")
+
+    # Remove any previously autogenerated API documentation.
+    doc_api_files = os.listdir(api_path)
+    permanent = ['chaco.api.rst', 'templates']
+    previously_autogenerated = \
+        [file for file in doc_api_files if file not in permanent]
+    for file in previously_autogenerated:
+        os.remove(os.path.join(api_path, file))
```

Comment:
This might be overkill but okay (not an actionable comment). I guess this is needed if we rename modules and dont do git clean locally between docs builds.